src/viem/estimateFeesPerGas.ts

```diff
@@ -17,14 +17,27 @@ export async function estimateFeesPerGas(
       getMaxPriorityFeePerGas(client, feeCurrency),
     ])
     const maxFeePerGas = gasPrice + maxPriorityFeePerGas
-    return { maxFeePerGas, maxPriorityFeePerGas }
+    // TODO: properly calculate baseFeePerGas
+    return { maxFeePerGas, maxPriorityFeePerGas, baseFeePerGas: BigInt(maxFeePerGas) }
```

Comment:
I'll address this in a follow up PR. We need to call the GasPriceMinimum contract to get the right value based on the fee currency set.

Reply:
Addressed in https://github.com/valora-inc/wallet/pull/4796